diffpy/pdfmorph/pdfmorph_api.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env python
+##############################################################################
+#
+# diffpy.pdfmorph   by DANSE Diffraction group
+#                   Simon J. L. Billinge
+#                   (c) 2018 Trustees of the Columbia University
+#                   in the City of New York.  All rights reserved.
+#
+# File coded by:    Timothy Liu
+#
+# See AUTHORS.txt for a list of people who contributed.
+# See LICENSE.txt for license information.
+#
+##############################################################################
+
+from collections import Iterable
+from diffpy.pdfmorph import morphs
+from diffpy.pdfmorph import refine as ref
+from diffpy.pdfmorph import tools
+
+import matplotlib.pyplot as plt
+
+default_config = dict(scale=None, stretch=None, smear=None,
+                      baselineslope=None, qdamp=None)
+
+# map of operation dict
+# TODO: include morphing on psize
+morph_step_dict = dict(scale=morphs.MorphScale,
+                       stretch=morphs.MorphStretch,
+                       smear=[morphs.MorphXtalPDFtoRDF,
+                              morphs.MorphSmear,
+                              morphs.MorphXtalRDFtoPDF],
+                       qdamp=morphs.MorphResolutionDamping)
```

Comment:
Are `default_config` and `morph_step_dict` constant?
If so maybe they should be in upper case or prefixed with `_` to mark them as module-internal variables.